example/TrueSDKSample/QATitleSelectionTableViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  QATitleSelectionTableViewController.swift
+//  TrueSDKSample
+//
+//  Created by Sreedeepkesav M S on 02/01/20.
+//  Copyright © 2020 Aleksandar Mihailovski. All rights reserved.
+//
+
+import UIKit
+import TrueSDK
+
+extension Notification.Name {
+    static let titleChanged = Notification.Name("loginTitleChanged")
+}
+
+private var titles : [String] {
```

Comment:
I think you should make it a private let inside the class

Reply:
Done